src/workers.rs

```diff
@@ -17,10 +18,10 @@ pub async fn start_workers() {
     queue.connect(NoTls).await.unwrap();
 
     let params = SleepParams {
-        sleep_period: 1,
-        max_sleep_period: 5,
-        min_sleep_period: 0,
-        sleep_step: 1,
+        sleep_period: Duration::from_secs(1),
+        max_sleep_period: Duration::from_secs(5),
```

Comment:
5 seconds may be too long

Reply:
[typing , SleepParams lower](https://github.com/pxp9/weather_bot_rust/pull/23/commits/343ef07a39250a164bfb8386946b90a08ab04494)